package.json

```diff
@@ -35,15 +36,16 @@
 		"babel-eslint": "^10.0.3",
 		"babel-jest": "^24.9.0",
 		"babel-loader": "^8.0.6",
-		"clean-webpack-plugin": "^3.0.0",
 		"core-js": "^3.2.1",
 		"css-loader": "^3.2.0",
 		"eslint": "^6.3.0",
 		"eslint-loader": "^3.0.0",
 		"jest": "^24.9.0",
 		"jest-cli": "^24.9.0",
 		"jest-transform-stub": "^2.0.0",
+		"mini-css-extract-plugin": "^0.9.0",
 		"node-sass": "^4.13.0",
+		"parallel-webpack": "^2.6.0",
```

Comment:
Is that the cause of the double clean error?

Reply:
No it doesn't matter if the two builds are done consecutively or concurrently -- there will be two writes to the directory regardless